Source/Response.swift

```diff
@@ -77,4 +77,16 @@ public extension Response {
         }
         return string
     }
+
+    /// Maps received data at key path into a String.
+    public func mapString(atKeyPath keyPath: String?) throws -> String {
```

Comment:
Hmm, here we don't have default value for `keyPath` parameter, but for RxSwift & ReactiveSwift extensions we have `nil` as default. I would rather stick to one, but I'm not sure which one is better, what do you think?